scripts/coverage.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+#
+# Runs all tests and collects code coverage
+#
+# Warning: this process is a little slow
+#
+
+set -e
+cd "$(dirname "$0")/.."
+
+_() {
+  echo "--- $*"
+  "$@"
+}
+
+: "${BUILDKITE_COMMIT:=local}"
+reportName="lcov-${BUILDKITE_COMMIT:0:9}"
+
+export RUSTFLAGS="
+  -Zprofile
+  -Zno-landing-pads
+  -Ccodegen-units=1
+  -Cinline-threshold=0
+  -Coverflow-checks=off
+"
+export CARGO_INCREMENTAL=0
+export RUST_BACKTRACE=1
+
+echo "--- remove old coverage results"
+
+if [[ -d target/cov ]]; then
+  find target/cov -name \*.gcda -print0 | xargs -0 rm -f
+fi
+rm -rf target/cov/$reportName
+
+_ cargo +nightly build --target-dir target/cov --all
+_ cargo +nightly test --target-dir target/cov --lib --all
+
+_ scripts/fetch-grcov.sh
+echo "--- grcov"
+./grcov target/cov/debug/deps/ > target/cov/lcov_full.info
+
+echo "--- filter_non_local_files_from_lcov"
```

Comment:
You should be able to use the `-s` option of grcov to achieve the same result (ignore any file which is not in the repository)

Reply:
Oh sweet, thanks.

Reply:
💔 
```bash
$ ./grcov target/cov/debug/deps/ -s . > out.lcov
thread 'main' panicked at 'Only one file in the repository should end with src/packet.rs (sdk/src/packet.rs and src/packet.rs both end with that)', src/path_rewriting.rs:143:13
note: Run with `RUST_BACKTRACE=1` for a backtrace.
```

Reply:
I'll fix this.

Reply:
Hey @marco-c, what's generally the best way to contact you?  Care to join our Discord forum? https://discord.gg/k8Kq58